website/docs/docs/collaborate/dbt-explorer-faqs.md

```diff
@@ -0,0 +1,176 @@
+---
+title: "dbt Explorer FAQs"
+sidebar_label: "dbt Explorer FAQs"
+description: "Learn more with the FAQs about dbt Explorer, how it works, how to interact with it, and more."
+pagination_next: null
+---
+
+[dbt Explorer](/docs/collaborate/explore-projects) is dbt Cloud’s new knowledge base and lineage visualization experience. It offers an interactive and high-level view of your company’s entire data estate, where you can do dive deeps for the context you need to understand and improve lineage so your teams can trust the data they’re using to make decisions.
+
+## Overview
+
```

Comment:
Should we have any intro text here?

Reply:
it follows the style for the other FAQ pages:

- https://docs.getdbt.com/docs/use-dbt-semantic-layer/sl-faqs
- https://docs.getdbt.com/best-practices/how-we-mesh/mesh-4-faqs

Reply:
No need to block it here, but I think that should change too. Not necessarily a blurb, but a word other than "Overview. " Something like General questions since it's an FAQ